### ISSUE TITLE
Unable to auth when adding the integration

### ISSUE THREAD

**Issue by julianvdhogen** [iMicknl/ha-nest-protect#333]
### The problem

when auth token the google page shows a 400 failure invalid_request
![Screenshot 2024-05-28 142618](https://github.com/iMicknl/ha-nest-protect/assets/10560914/5c38ed0f-71ed-4c4c-b0d8-de5d298ba0f6)

### What version of this integration (ha-nest-protect) has the issue?

0.3.12

### What version of Home Assistant Core has the issue?

Core 2024.5.4

### Device / Model

Nest protect battery v2

### Home Assistant log

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks (`)
```
</details>

**Comment by evertterpstra** [comment]
![image](https://github.com/iMicknl/ha-nest-protect/assets/123770708/6d369814-d619-49c0-aded-f844d5d7ba7e)
Same problem here........................

**Comment by evertterpstra** [comment]
![image](https://github.com/iMicknl/ha-nest-protect/assets/123770708/e07e7492-9065-4355-826e-07f3e38d4c59)

**Comment by iMicknl** [comment]
Duplicate of #105 (pinned issue..).

TLDR: use beta version
